lib/snapshot-eviction/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
```

Comment:
Seeing the formatting here has led me to discover that a `:check-format-toml` rule isn't being added via the `rust_library` macro, which is my fault. This `Cargo.toml` is fine given the current CI checks so I can update when the rule is in place (it's mostly around sorting deps lexically and using the workspace-provided metadata from the root)